ghcup-cross-0.0.8.yaml

```diff
@@ -138,3 +138,16 @@ ghcupDownloads:
               dlOutput: ghc-9.10.0.20240412-x86_64-linux-alpine3_18-wasm-cross_wasm32-wasi-release+fully_static.tar.xz
           Linux_Alpine:
             unknown_versioning: *ghc-wasm32-wasi-9101-a3-64-static
+    wasm32-wasi-9.10.1:
```

Comment:
Is this cut from the *true* 9.10.1 branch?

Reply:
Almost, the [pipeline for the `ghc-9.10.1-release` tag](https://gitlab.haskell.org/ghc/ghc/-/pipelines/94655) does not contain a WASM bindist, so instead, the [job](https://gitlab.haskell.org/ghc/ghc/-/jobs/1892418) that produced this bindist is [`934a8bb930390d5db1e5613ddfeafc8c80dd9b96`](https://gitlab.haskell.org/ghc/ghc/-/commit/934a8bb930390d5db1e5613ddfeafc8c80dd9b96), the current head of the `ghc-9.10` branch. The only difference to `ghc-9.10.1-release` is this MR: https://gitlab.haskell.org/ghc/ghc/-/merge_requests/12716